helpers/file.go

```diff
@@ -0,0 +1,13 @@
+package helper
+
+import "os"
+
+func CheckFileExists(path string) bool {
+	_, err := os.Stat(path)
+	return !os.IsNotExist(err)
+}
+
+func CheckFileNotExists(path string) bool {
+	_, err := os.Stat(path)
+	return os.IsNotExist(err)
+}
```

Comment:
Why do you need 2 identical functions just by reversing? If you use CheckFileExists which returns boolean, you could use NOT CheckFileExists which still returns boolean but reversed.

Maybe rename as well, like CheckFileExists → CheckIfFileExists

Reply:
Agree, fixed in [db9a3bf](https://github.com/LookinLabs/GoBlitz/pull/21/commits/db9a3bf1cf360aaba5483cc75761ef4795fc5c48)